plugins/modules/azure_rm_storageaccount.py

```diff
@@ -17,6 +17,17 @@
 description:
     - Create, update or delete a storage account.
 options:
+    auth_mode:
+        description:
+            - The mode in which to run the command. C(login) mode will directly use your login credentials for the authentication.
```

Comment:
@mucsi96 Instead of calling this a "login" mode, can we rename it to "aad" (preferable) or "entra_id", to make it more clear to the users?